shared/js/background/safari-events.es6.js

```diff
@@ -143,13 +145,25 @@ let getSetting = (e) => {
 
     let setting = JSON.parse(JSON.stringify(settings.getSetting(name)))
 
-    // Safari optons page has to send a message to the background
-    // and includes an id to help identify the correct response
-    setting.id = e.message.id
```

Comment:
Turns out this was erroring out in Safari. `setting` is a string here, so setting properties on it breaks.